### DISCUSSION TITLE
Implementing Radioss card with model created with LS-Dyna

### DISCUSSION THREAD

**Discussion by FGarelli** [OpenRadioss/OpenRadioss#2498] [Q&A]
Dear all, I'm currently using LS-Prepost to prepare the models to be runned in Openradioss but I would like to use a material card which is not translated in the starter file by Radioss. 
How can I introduce a Radioss card to a model prepared in LS-Dyna?

**Comment by elequiniou** [comment]
Hi FGarelli, 
Please, did you give a try with *INCLUDE_RADIOSS to use an include file in Radioss format ?
It should be understood by OpenRadioss. Hope it helps.

**Comment by FGarelli** [comment]
Thank you. This will solve the problem.
For the model I want to use thick shell element which are not supported in the LS-DYNA conversion. I was planning to use the same methodology converting the thick shell model in Radioss format but I get this error when I run the code.
![image](https://github.com/OpenRadioss/OpenRadioss/assets/132582318/b22c8957-1941-4617-9d00-a5838d7ca86c)
Is it a problem of conversion?
I attached the input file that I'm using.
[tshell.zip](https://github.com/OpenRadioss/OpenRadioss/files/14978633/tshell.zip)

**Comment by sebastienVilleneuve** [comment]
Hi,

In the *0.rad mode that is included , you need a 'valid' Radioss model , with #RADIOSS STARTER at first line + /BEGIN option : 

![image](https://github.com/OpenRadioss/OpenRadioss/assets/100404030/8f42ef22-ad59-4610-83f4-5d956e1e2615)

You also need /END at last 0.rad file line.

An other remark, MIXED Radioss / Dyna need PART & associated mat & prop in the same format.

Ragards,
Sébastien

**Reply by FGarelli** [reply]
Hi Sebastian, thank you for the information.
So the use of *INCLUDE_RADIOSS is limited just for boundary conditions and load because I cannot define a part (node, elements, materiale, prop) with a mix of Dyna and radioss, correct?
I thought that openradioss was traslating Dyna in Radioss so it would have been necessary to use *INCLUDE_RADIOSS just for the missing card that are able to be translated at the moment (https://2022.help.altair.com/2022.1/hwsolvers/rad/topics/solvers/rad/ls_dyna_keywords_r.htm)
Where I didn't understand it correctly?

**Reply by FGarelli** [reply]
If creating a mixed model is not possible, does it exist a script to translate the LS-DYNA cards in the Radioss format to be able to create the model with LS-Prepost and add the missing cards directly on the code?

**Comment by sebastienVilleneuve** [comment]
Hi,

Maybe my explanation was no clear enough:
It is possible to mix format, but you need a Radioss include that respect the Radioss format ( #RADIOSS STARTER + /BEGIN + /END inside)
+ additional constraint that a given part + concerned material + prop in a same format)

Can you give us the original .k file ? ( without Raidoss include) 

Thanks,
Sébastien

**Reply by FGarelli** [reply]
Okay, thank you for the clarification. Here is the the original file.
[sub_tshell_ls_dyna.zip](https://github.com/OpenRadioss/OpenRadioss/files/14980436/sub_tshell_ls_dyna.zip)

**Comment by sebastienVilleneuve** [comment]
Hi,

Did you try with last OpenRadioss build ?

It seems to be ok ;
![image](https://github.com/OpenRadioss/OpenRadioss/assets/100404030/82f3a0dc-2090-4492-80fd-044ccff03732)

Sébastien

**Reply by FGarelli** [reply]
I didn't try. You run the file as I shared?

**Reply by FGarelli** [reply]
With the updated version I obtain this error: 
![image](https://github.com/OpenRadioss/OpenRadioss/assets/132582318/462028f5-9bfe-4920-aa43-be339190719d)

**Reply by FGarelli** [reply]
Hello, I managed to run the model with the updated code but I get this error
![image](https://github.com/OpenRadioss/OpenRadioss/assets/132582318/0d7c5ce8-3ea7-40c7-933b-3fa393422c69)
I attached the output from the starter.
[sub_tshell_ls_dyna_0000.zip](https://github.com/OpenRadioss/OpenRadioss/files/15013283/sub_tshell_ls_dyna_0000.zip)

**Comment by sebastienVilleneuve** [comment]
Hi,

Is it possible to have the model ? is it the one you already attached ?
( I can not reproduce the issue)

Thanks,
Sébastien

**Reply by FGarelli** [reply]
Hi,
Yes, the model is the same that I attached before.
Is it possible that I didn't update correctly the last version of openradioss?
